### DISCUSSION TITLE
Why is this happening?

### DISCUSSION THREAD

**Discussion by zhanxianchanghe** [geodynamics/sw4#224] [General]
I submitted a sw4 task to the server for calculation. Why did the calculation stop immediately after reading the files required for calculation?
![微信截图_20241029213020](https://github.com/user-attachments/assets/11bd65f1-eb2b-4eb6-95d0-80794379ae55)

**Comment by houjun** [comment]
It seems your job’s wall clock time (4 minutes) is too short, and SW4 didn’t reach the simulation start phase.
